agile-development/collaboration/readme.md

```diff
@@ -69,3 +69,8 @@ As soon the pair finds out that the PBI will warrant swarming, the pair brings i
 ## Benefits of increased collaboration
 
 Knowledge sharing and bringing CSE and customer engineers together in a ‘code-with’ manner is an important aspect of CSE engagements. This grows both our customers’ and our CSE team’s capability to build on Azure. We are responsible for demonstrating engineering fundamentals and leaving the customer in a better place after we disengage. This can only happen if we collaborate and engage together as a team. In addition to improved software quality, this also adds a beneficial social aspect to the engagements.
+
+## Resources
+
```

Comment:
Looks good, can you please add a note saying that these are examples and can be adapted to each project? 
In the second link, there is a list of fixed time duration that might not fit all projects.

Thanks!

Reply:
Sure, on it.

Reply:
Also, the time duration is a pattern which we tested with some teams (including customer and internal), but can be adapted as well, of course

Reply:
@meulta Is the change reflecting what you had in mind?

Reply:
Yep thanks!!